app/scripts/modules/cluster/reports/forms/cluster.project.form.details.js

```diff
@@ -759,6 +781,43 @@ angular.module( 'ngm.widget.project.details', [ 'ngm.provider' ])
 						}
 					});
 
+					//SAVE COMPONENT-PLANS
+					
+					 $scope.project.definition.plan_component = [];
+
+					 	if ($scope.project.definition.hrp_plan == true) {
+						  $scope.project.definition.plan_component.push('hrp_plan');
+						}
```

Comment:
cool as array of _ids but what if use common RH structure as array of json objects with _id and _name?

Reply:
any possible variation for example

project_plans:[{
    plan_id:
    plan_name: 
    plan_components:[{component_id: component_name:}] 
}]

or if flat in case sails does not support deep "project_pleans.plan_components.component_id" search

project_plans:[{
    // type: "plan"
    plan_id:
    plan_name:
},{
    // type: "component"
    component_id:
    component_name: 
    plan_id:
    plan_name: 
}]

and filter on backend
"project_plans.component_id" and "project_plans.plan_id"